xmtp/src/utils.rs

```diff
@@ -30,3 +32,7 @@ pub fn build_envelope(content_topic: String, message: Vec<u8>) -> Envelope {
 pub fn base64_encode(bytes: &[u8]) -> String {
     general_purpose::STANDARD_NO_PAD.encode(bytes)
 }
+
+pub fn key_fingerprint(key: &Curve25519PublicKey) -> String {
+    base64_encode(keccak256(key.to_string().as_str()).as_slice())
+}
```

Comment:
This makes the base64 encoded hash of a publickKey a named operation which can be used form multiple files consistently